pkg/api/v1/common/common.go

```diff
@@ -51,6 +52,13 @@ func (rn *ResourceRefNamespaced) GetObject(parentNamespace string) *client.Objec
 	return &key
 }
 
+func (rn *ResourceRefNamespaced) GetNamespacedName() types.NamespacedName {
```

Comment:
Why not use `ResourceRefNamespaced.GetObject()` instead?

Reply:
Done - but seems weird that GetObject takes a namespace param when the object it's operating on already has one?